models/sdn-fabric-0.1.x/sdn-fabric.tree

```diff
@@ -0,0 +1,60 @@
+module: onf-switch
+  +--rw switch* [switch-id]
+     +--rw switch-id             inet:domain-name
+     +--rw display-name?         string
+     +--rw description?          string
+     +--rw attribute* [attribute-key]
+     |  +--rw attribute-key    yt:yang-identifier
+     |  +--rw value            string
+     +--rw model                 -> /sm:switch-model/switch-model-id
+     +--rw role                  enumeration
+     +--rw management
+     |  +--rw address?       inet:ipv4-address-no-zone
+     |  +--rw port-number?   inet:port-number
+     +--rw port* [cage-number channel-number]
```

Comment:
Should there be per-port opstate in here as well?
    observed-speed
    connected [up | down]
    ...

Reply:
I have added both these in